app/src/main/java/com/lastaoutdoor/lasta/data/db/SocialDBRepositoryImpl.kt

```diff
@@ -119,17 +180,13 @@ class SocialDBRepositoryImpl @Inject constructor(context: Context, database: Fir
   }
 
   override suspend fun sendFriendRequest(userId: String, receiverId: String): Boolean {
-    var success = false
+    var success = true

```

Comment:
What does this variable represent ? Maybe tell the other devs through a comment line

Reply:
Okay I took care of it !